src/ensembl/src/content/app/entity-viewer/shared/components/entity-viewer-sidebar/entity-viewer-sidebar-modal/modal-views/EntityViewerBookmarks.tsx

```diff
@@ -86,6 +79,18 @@ export const PreviouslyViewedLinks = (props: PreviouslyViewedLinksProps) => {
   );
 };
 
+const buildLabelElement = (labelText: string | string[]) => {
```

Comment:
Not sure, but can we move this to a shared helper function?

Reply:
Moved the recurring bit of rendering logic into a dedicated TextLine component.